src/test/java/com/commercetools/project/sync/cartdiscount/CartDiscountSyncerTest.java

```diff
@@ -54,35 +88,34 @@ void transform_ShouldReplaceCartDiscountReferenceIdsWithKeys() {
     assertThat(draftsFromPageStage).isCompletedWithValue(expectedResult);
   }
 
-  @Test
-  void getQuery_ShouldBuildCartDiscountQuery() {
```

Comment:
What about this test?

Reply:
I removed it because this is already tested in `of_ShouldCreateCartDiscountSyncerInstance`.